values/gatekeeper-artifacts/gatekeeper-artifacts.gotmpl

```diff
@@ -25,6 +25,6 @@ config:
     #   kind: "Service"
   # Exempted Namespaces
   match:
-    - excludedNamespaces: ["gatekeeper-system"]
+    - excludedNamespaces: ["gatekeeper-system", "kube-system"]
```

Comment:
This should be configurable (with these two hardcoded). I see companies wanting a sandbox to try stuff in and not be limited by gatekeeper. Example:
```yaml
- excludedNamespaces: {{ list "gatekeeper-system", "kube-system" | append ($v.charts.gatekeeper | get "excludedNamespaces" list) | toYaml | nindent 8 }}
```

Reply:
Ok, sounds reasonable. 
Just to point out that in `gotmpl`, appending a list to other list does not flatten the items and it ends up as nested item inside the list. Concat on the other hand allows you to join lists.

Reply:
great, that one indeed